tools/vsimporter/src/vsimporter.cpp

```diff
@@ -282,15 +290,11 @@ int main(int argc, char* argv[])
   // Initialize global settings
   String binaryDir = sb_dirname(getBinaryLocation());
   sbValidateWithTelemetry(!binaryDir.empty(), "Failed to resolve binary directory.");
+  if (!templatesPath.empty()) {
+    settingsManager.setGlobalVar("VSIMPORTER_TEMPLATES_DIR", joinPaths(getcwd(), templatesPath));
```

Comment:
My workflow was was running vsimporter.exe from the xcodeproj dir. So it was relative to that location. I think it makes a little more sense to have it relative to the binary but there's a few options here. Preferably it would be run with the full path to the templates, not a relative one.